src/apps/competitions/unpacker/exceptions.py

```diff
@@ -0,0 +1,7 @@
+
+class CompetitionUnpackingException(Exception):
+    pass
+
+
+class CompetitionConversionException(CompetitionUnpackingException):
```

Comment:
is this useful? we don't even raise it anywher eI don't see?

Reply:
Converter line 214, 257, 259. Should probably be moved with the refactor